packages/embark/src/test/events.js

```diff
@@ -29,6 +30,7 @@ describe('embark.Events', function () {
 
     it('should be able to listen to an event emission once', (done) => {
       events.once(testEventName, ({isTest}) => {
+        // eslint-disable-next-line no-unused-expressions
```

Comment:
Just an FYI, you can disable eslint rules for whole files by putting the following at the top of the file
```
/* eslint no-unused-expressions: 0 */
```
It's especially useful in tests

Reply:
Ah, very nice, will keep that in mind for the future. Also, it will get cleaned up when those tests are eventually migrated to jest.